src/dbg/commands/cmd-analysis.cpp

```diff
@@ -1,4 +1,5 @@
 #include "cmd-analysis.h"
+#include "ntdll/ntdll.h"

```

Comment:
No, there are two alternatives:
- `#include <ntstatus.h>`
  `#include "cmd-analysis.h`
  `// all other includes`
- `// all includes`
  `#undef WIN32_NO_STATUS`
  `#include <ntstatus.h>`

Same for the other .cpp. I think the second option (`#undef` followed by `#include`) is pretty ugly. I chose this over the first option because that way the file's own header would still be first. But any of the three works.